### PR TITLE
Split and move process commands

### PR DESCRIPTION
**Pull request by openbots-su**

### CODE REVIEW

File: OpenBots.Commands/OpenBots.Commands.SystemAutomation/OpenBots.Commands.Process/RunPowershellScriptCommand.cs [hunk 1]
```diff
@@ -54,8 +43,6 @@ public RunScriptCommand()
 			SelectionName = "Run Script";
```
Comment (openbots-ff) [inline OpenBots.Commands/OpenBots.Commands.SystemAutomation/OpenBots.Commands.Process/RunPowershellScriptCommand.cs:43]:
```suggestion
			SelectionName = "Run PowerShell Script";
```

Comment (openbots-ff) [reply OpenBots.Commands/OpenBots.Commands.SystemAutomation/OpenBots.Commands.Process/RunPowershellScriptCommand.cs:43]: I can't add a comment to the line above but it should be RunPowerShellScriptCommand. If you even try to load these commands in studio, you should see a repeat of their names in the commands tab.

File: OpenBots.Commands/OpenBots.Commands.SystemAutomation/OpenBots.Commands.Process/RunPowershellScriptCommand.cs [hunk 2]
```diff
@@ -11,12 +11,11 @@
...
 using System.Windows.Forms;
 using Diagnostics = System.Diagnostics;
-
 namespace OpenBots.Commands.Process
 {
-	[Serializable]
+    [Serializable]
 	[Category("Programs/Process Commands")]
-	[Description("This command runs a script or program and waits for it to exit before proceeding.")]
+	[Description("This command runs a Powershell script or program and waits for it to exit before proceeding.")]
```
Comment (openbots-ff) [inline OpenBots.Commands/OpenBots.Commands.SystemAutomation/OpenBots.Commands.Process/RunPowershellScriptCommand.cs:18]:
```suggestion
	[Description("This command runs a PowerShell script or program and waits for it to exit before proceeding.")]
```

File: OpenBots.Commands/OpenBots.Commands.SystemAutomation/OpenBots.Commands.Process/RunPowershellScriptCommand.cs [hunk 3]
```diff
@@ -113,15 +76,14 @@ public override List<Control> Render(IfrmCommandEditor editor, ICommandControls
...
-			RenderedControls.AddRange(commandControls.CreateDefaultDropdownGroupFor("v_ScriptType", this, editor));
 			RenderedControls.AddRange(commandControls.CreateDefaultInputGroupFor("v_ScriptArgs", this, editor));
 
 			return RenderedControls;
 		}
 
 		public override string GetDisplayValue()
 		{
-			return base.GetDisplayValue() + $" [Script Path '{v_ScriptPath}']";
+			return base.GetDisplayValue() + $" [Powershell Script Path '{v_ScriptPath}']";
```
Comment (openbots-ff) [inline OpenBots.Commands/OpenBots.Commands.SystemAutomation/OpenBots.Commands.Process/RunPowershellScriptCommand.cs:86]:
```suggestion
			return base.GetDisplayValue() + $" [PowerShell Script Path '{v_ScriptPath}']";
```

File: OpenBots.Commands/OpenBots.Commands.SystemAutomation/OpenBots.Commands.Process/RunPythonScript.cs [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+		[DisplayName("Arguments (Optional)")]
+		[Description("Enter any arguments as a single string.")]
+		[SampleUsage("-message Hello -t 2 || {vArguments}")]
+		[Remarks("This input is optional.")]
+		[Editor("ShowVariableHelper", typeof(UIAdditionalHelperType))]
+		public string v_ScriptArgs { get; set; }
+
+		public RunPythonScriptCommand()
+		{
+			CommandName = "RunScriptCommand";
```
Comment (openbots-ff) [inline OpenBots.Commands/OpenBots.Commands.SystemAutomation/OpenBots.Commands.Process/RunPythonScript.cs:45]:
```suggestion
			CommandName = "RunPythonScriptCommand";
```

File: OpenBots.Commands/OpenBots.Commands.SystemAutomation/OpenBots.Commands.Process/RunPythonScript.cs [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+			SelectionName = "Run Script";
```
Comment (openbots-ff) [inline OpenBots.Commands/OpenBots.Commands.SystemAutomation/OpenBots.Commands.Process/RunPythonScript.cs:46]:
```suggestion
			SelectionName = "Run Python Script";
```

**Review by openbots-ff — CHANGES_REQUESTED** [review]

File: OpenBots.Commands/OpenBots.Commands.SystemAutomation/OpenBots.Commands.Process/RunPythonScript.cs (hunk 5 above)
Comment (openbots-su) [reply OpenBots.Commands/OpenBots.Commands.SystemAutomation/OpenBots.Commands.Process/RunPythonScript.cs]: Addressed in 2f90f4c

File: OpenBots.Commands/OpenBots.Commands.SystemAutomation/OpenBots.Commands.Process/RunPythonScript.cs (hunk 4 above)
Comment (openbots-su) [reply OpenBots.Commands/OpenBots.Commands.SystemAutomation/OpenBots.Commands.Process/RunPythonScript.cs]: Addressed in 2f90f4c

File: OpenBots.Commands/OpenBots.Commands.SystemAutomation/OpenBots.Commands.Process/RunPowershellScriptCommand.cs (hunk 3 above)
Comment (openbots-su) [reply OpenBots.Commands/OpenBots.Commands.SystemAutomation/OpenBots.Commands.Process/RunPowershellScriptCommand.cs:86]: Addressed in 2f90f4c

File: OpenBots.Commands/OpenBots.Commands.SystemAutomation/OpenBots.Commands.Process/RunPowershellScriptCommand.cs (hunk 1 above)
Comment (openbots-su) [reply OpenBots.Commands/OpenBots.Commands.SystemAutomation/OpenBots.Commands.Process/RunPowershellScriptCommand.cs]: Addressed in 2f90f4c

File: OpenBots.Commands/OpenBots.Commands.SystemAutomation/OpenBots.Commands.Process/RunPowershellScriptCommand.cs (hunk 2 above)
Comment (openbots-su) [reply OpenBots.Commands/OpenBots.Commands.SystemAutomation/OpenBots.Commands.Process/RunPowershellScriptCommand.cs:18]: Addressed in 2f90f4c

**Review by openbots-ff — APPROVED** [review]

**Review by openbots-ch — APPROVED** [review]

**Pull request merged**